constant.go

```diff
@@ -1,6 +1,6 @@
-// The MIT License (MIT)
+// MIT License
 //
-// Copyright (c) 2016 Bo-Yi Wu
+// Copyright (c) 2019 Gin-Gonic
```

Comment:
What is the context for this change?

Reply:
The previous github action did not check for problems, this is a match to the contents of .licenserc.yaml